Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -151,7 +151,9 @@ public class MapboxNavigationService: NSObject, NavigationService {
     /**
      A reference to a MapboxDirections service. Used for rerouting.
      */
-    public var directions: Directions
+    public var directions: Directions {
+        NavSDK.directions
```

Comment:
There is a case when you want to request a route from one endpoint and also use it for rerouting, but use a different endpoint for navigation tiles(in Navigator)

Reply:
I decided to keep previous implementation and just default to the NavSDK.directions instead of simply .shared.

Reply:
Done, please review the latest revision.